src/aro/Type.zig

```diff
@@ -425,6 +425,7 @@ data: union {
 specifier: Specifier,
 qual: Qualifiers = .{},
 decayed: bool = false,
+statically_sized_array: bool = false,
```

Comment:
`static_array` specifier exists but is seemingly unused currently. Could you try using that instead?

(Apparently it was only used for three days between c35d5c22f7adc314202c5dcd220d88d6ff7d068d and 9b4daaa8647d8ea7c24ed6edae66a32433fb4ba6)